Checkout/Source/Tokenisation/CheckoutAPIService.swift

```diff
@@ -170,3 +171,53 @@ final public class CheckoutAPIService: CheckoutAPIProtocol {
     }
   }
 }
+
+// MARK: Security Code Request
+
+extension CheckoutAPIService {
+  public func createSecurityCodeToken(securityCode: String, completion: @escaping (Result<SecurityCodeTokenDetails, TokenisationError.SecurityCodeError>) -> Void) {
+    guard !publicKey.isEmpty else {
+        completion(.failure(.missingAPIKey))
+        return
+    }
+
+    let request = SecurityCodeRequest(tokenData: TokenData(securityCode: securityCode))
+    let requestParameterResult = requestFactory.create(request: .securityCodeToken(request: request, publicKey: publicKey))
+
+    switch requestParameterResult {
+    case .success(let requestParameters):
+//      logManager.queue(event: .tokenRequested(CheckoutLogEvent.TokenRequestData(
```

Comment:
Is this intentionally commented ?

Reply:
yeah, to mark where we will log it and how and make me remember during the logging ticket work

Reply:
removed this line